modules/webgl/src/context/context.js

```diff
@@ -114,6 +101,13 @@ export function createGLContext(options = {}) {
 }
 
 export function instrumentGLContext(gl, options = {}) {
+  // Avoid multiple instrumentations
+  if (gl._instrumented) {
```

Comment:
Did you notice and issue related to this?

Reply:
Yeah, I have seen this while running the tests because animationloop does explictly instruments the provided gl context :  https://github.com/uber/luma.gl/blob/006e02d1c3b17e27580ab2d0eb1599b30fa4938b/modules/core/src/lib/animation-loop.js#L400